backend/taxjarApi.ts

```diff
@@ -0,0 +1,82 @@
+import Taxjar from "taxjar";
+import { TaxForOrderRes } from "taxjar/dist/util/types";
+import { OrderSubscriptionFragment } from "../generated/graphql";
+import { CheckoutPayload, FetchTaxesPayload, TaxJarConfig } from "./types";
+
+export const fetchTaxes = async (
+  taxParams: FetchTaxesPayload,
+  taxJarConfig: TaxJarConfig
+) => {
+  const client = new Taxjar({
+    apiKey: taxJarConfig.apiKey,
+    apiUrl: taxJarConfig.sandbox
+      ? Taxjar.SANDBOX_API_URL
+      : Taxjar.DEFAULT_API_URL,
+  });
+  const response: TaxForOrderRes = await client.taxForOrder({
+    from_country: taxJarConfig.shipFrom.fromCountry,
+    from_zip: taxJarConfig.shipFrom.fromZip,
+    from_state: taxJarConfig.shipFrom.fromState,
+    from_city: taxJarConfig.shipFrom.fromCity,
+    from_street: taxJarConfig.shipFrom.fromStreet,
+
+    to_country: taxParams.address.country,
+    to_zip: taxParams.address.postal_code,
+    to_state: taxParams.address.country_area,
+    to_city: taxParams.address.city,
+    to_street: `${taxParams.address.street_address_1} ${taxParams.address.street_address_2}`,
+    shipping: Number(taxParams.shipping_amount),
+    line_items: taxParams.lines.map((line) => ({
+      id: line.id,
+      quantity: line.quantity,
+      product_tax_code:
+        line.product_metadata.taxjar_tax_code ||
+        line.product_type_metadata.taxjar_tax_code,
+      unit_price: Number(line.unit_amount),
+      // FIXME: take into account a discount that we recieve in CheckoutPayload
+      discount: 0,
+    })),
+  });
+  return response;
+};
+
+export const createOrderTransaction = async (
+  order: OrderSubscriptionFragment,
+  taxJarConfig: TaxJarConfig
+) => {
+  const client = new Taxjar({
+    apiKey: taxJarConfig.apiKey,
+    apiUrl: taxJarConfig.sandbox
+      ? Taxjar.SANDBOX_API_URL
+      : Taxjar.DEFAULT_API_URL,
+  });
```

Comment:
Since you're reusing this twice, you can make a function:
```
const getTaxjarClient = (taxJarConfig: TaxJarConfig) => new Taxjar({
    apiKey: taxJarConfig.apiKey,
    apiUrl: taxJarConfig.sandbox
      ? Taxjar.SANDBOX_API_URL
      : Taxjar.DEFAULT_API_URL,
  })
```